tmk_core/common/keyboard.c

```diff
@@ -62,24 +62,47 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #endif
 
 
-
 #ifdef MATRIX_HAS_GHOST
-static bool has_ghost_in_row(uint8_t row)
+extern const uint16_t keymaps[][MATRIX_ROWS][MATRIX_COLS];
+static matrix_row_t get_real_keys(uint8_t row, matrix_row_t rowdata){
+    matrix_row_t out = 0;
+    for (int col = 0; col < MATRIX_COLS; col++) {
+        if (pgm_read_byte(&keymaps[0][row][col]) && (rowdata & (1<<col))){
+            out |= 1<<col;
+        }
+    }
+    return out;
+}
+
+static inline bool countones(matrix_row_t row)
```

Comment:
The implementation looks good now, but the function name is a bit misleading. Maybe something like `is_more_than_one_pressed` would be a better name. But if you can think of an even more descriptive and perhaps shorter name, feel free to change it to that instead.